### PR TITLE
Merge dev to master

### PR DESCRIPTION
**Pull request by totop716**

### CODE REVIEW

**Review by sameepsi — APPROVED** [review]

**Review by AamirAlam — APPROVED** [review]
Changes look good

File: package.json [hunk 1]
```diff
@@ -101,7 +101,11 @@
     "ejs": "^3.1.7",
     "cross-fetch": "^3.1.5",
     "eventsource": "^2.0.2",
-    "protobufjs": "^6.11.3"
+    "protobufjs": "^6.11.3",
+    "shell-quote": "^1.7.3",
```
Comment (FriskyHamTitz) [inline package.json:105]: Where were these 3 dependencies added

File: src/layouts/PageLayout.tsx [hunk 2]
```diff
@@ -28,8 +29,44 @@ const PageLayout: React.FC<PageLayoutProps> = ({ children, name }) => {
     };
   }, [history, updateIsProMode]);
 
+  useEffect(() => {
+    if (
+      window.location.host !== 'quickswap.exchange' &&
```
Comment (FriskyHamTitz) [inline src/layouts/PageLayout.tsx:34]: Are you sure the pass won't appear on production?  I need you to verify and test 100% before we merge

File: package.json (hunk 1 above)
Comment (FriskyHamTitz) [reply package.json:105]: I didn't see any new import statements with these why do we need it

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]

**Review by FriskyHamTitz — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (totop716) [reply package.json:105]: This is to fixing errors and warnings when running `yarn audit`

**Review by sameepsi — APPROVED** [review]

**Pull request merged**